portfolio/src/main/webapp/gallary.html

```diff
@@ -22,7 +22,20 @@
     </header>
         <!------gallary-->
         <div class="home" id="home">
-            <div class="home-text"><h1><span>Rockies in 2022</span></h1></div>
+            <div class="home-text">
+                <h1><span>Rockies in 2022</span></h1>
+                
+                <div class="upload">
+                    <form action="/upload" method="POST" enctype="multipart/form-data">
+                        <p>Want to show your mountain photos?</p>
+                        <p>Upload an image:</p>
+                        <input type="file" name="image"><br/>
+                        <button>Submit</button><br/>
+                    </form>
+                    
+                    <p>Click <a href="/list-images">here</a> to see all of the uploaded images!</p>
```

Comment:
Do think about the potential issues with showing all the images you uploaded. Don't need to do anything, but think about the cost of brining every images you uploaded from the server, what you could do to improve it etc.